widgets/common/src/DeploymentActionsMenu.jsx

```diff
@@ -32,13 +33,17 @@ export default function DeploymentActionsMenu({ onActionClick, toolbox, trigger
         content: translate(item.name),
         disabled: !isUserAuthorized(item.permission, managerState)
     }));
+    const popupMenuProps = !trigger
+        ? { bordered: true, help: i18n.t('widgets.common.deployments.actionsMenu.tooltip'), offset: [0, 5] }
+        : {};
 
     function onItemClick(event, { name }) {
         onActionClick(name);
     }
 
     return (
-        <PopupMenu className="deploymentActionsMenu">
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        <PopupMenu className="deploymentActionsMenu" {...popupMenuProps}>
```

Comment:
If we are justifying props spreading in such a case then what are the cases where it would be not justified?
We are disabling this rule constantly... how about disabling it globally?

Reply:
🙂 I think the rule in general makes sense - whenever it is possible we should be very clear in what we are passing. As in the rule description: _"This enhances readability of code by being more explicit (...) It is also good for maintainability by avoiding passing unintentional extra props"_.

In this particular case:
1. I found it a bit cleaner to do it like I did instead of:
```suggestion
        <PopupMenu
            className="deploymentActionsMenu"
            bordered={trigger ? undefined : true}
            help={trigger ? undefined : i18n.t('widgets.common.deployments.actionsMenu.tooltip')}
            offset={trigger ? undefined : [0, 5]}
        >
```
2. We exactly know what we are spreading, because `popupMenuProps` definition is 10 lines above.

What do you think? I'm open to any suggestions, but in general I'm for keeping the rule.

Reply:
I agree props spreading here is justified, I'm not against its usage here.
I just noticed we are disabling this rule frequently. At the same time I can see only one edge case where it would be unjustified: a scenario where props object is used only in one component and there is at most one conditional prop.
I think having this rule enabled globally actually discourages using this nice feature in cases where it would be justified and beneficial (actually this is my real experience). So I think it actually makes more harm then benefits. After all we have very picky code reviews, so if props spreading was used where it shouldn't it wouldn't be unnoticed.

Reply:
I think I'm tending to agree with your approach. As long as we have agreement on what are good and what are the bad uses of the props spreading, I think we could disable it globally. Let's discuss on our slack channel, OK? I'll start a thread and resolve this comment here (I would like not to change the rule setting within this PR). Hope that's fine with you.